iron/distribution.yaml

```
@@ -4895,6 +4895,24 @@ repositories:
       url: https://github.com/ros2/ros_testing.git
       version: iron
     status: maintained
+  ros_tutorials:
+    doc:
+      type: git
+      url: https://github.com/ros/ros_tutorials.git
+      version: iron
+    release:
+      packages:
+      - turtlesim
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros2-gbp/ros_tutorials-release.git
+      version: 1.6.2-1
```

Sorry, this was a typo in my previous suggestion:

```suggestion
      version: 1.6.1-1
```